### PR TITLE
[Preliminary] Add Drawer, Slide, Portal and Backdrop components

### PR DESCRIPTION
**Pull request by amarinov**
This is a preliminary pull request.
Unit tests are not yet ready.

### CODE REVIEW

**Comment by connor-baer** [comment]
Would it be possible to split this PR into multiple ones (maybe by component)?

**Comment by amarinov** [comment]
I could leave just the new lowest level components. Then make a new PR using this one as a base and  add on top the depending components, etc.

I am on it, but this is going to take me some time.

File: src/components/Popover/Popover.js [hunk 1]
```diff
@@ -125,14 +179,42 @@ class Popover extends Component {
...
+      modifiers,
+      usePortal,
+      ...others
     } = this.props;
 
+    const popper = isOpen && (
+      <Popper
+        {...others}
+        placement={toPopperPlacement(position, align)}
+        modifiers={{ ...modifiers, popperModifiers }}
```
Comment (elanoism) [inline src/components/Popover/Popover.js]: Need to destruct `popperModifiers` as well.

**Comment by amarinov** [comment]
Closing in favor of #269

**Pull request closed without merging**